### PR TITLE
fixes #30 - removes split view button from sidebar

### LINKED ISSUE #30: Remove editor split button from sidebar
Clicking on the split button and selecting "No Split" and then "Vertical Split" or "Horizontal Split" breaks the live preview and there is no way to get the preview back. Remove this button to prevent user from breaking the live preview.

### PR DESCRIPTION
**Pull request by K-LV**
fixes #30 - Prevents user from breaking the live preview from the split view button

### CODE REVIEW

**Comment by humphd** [comment]
screen shot 2015-03-12 at 10 46 01 am

I wonder if removing this is enough, or if we should do a bit more.  In the above screenshot (which doesn't have your patch), you'll see that we also have a `Left` and `Right` section.  It might make sense to remove these?

**Comment by K-LV** [comment]
@humphd I'll look into it.

**Comment by sedge** [comment]
That's appearing because we're hijacking the split pane functionality. I suspect changing this will be messy. Could there be a cleaner way to inject the iframe that doesn't confuse the editor like this?

**Comment by humphd** [comment]
Not without also doing major work to make our iframe some kind of a recognized editor document in Brackets.  That _might_ be worth doing upstream, but it will take forever to land ime.  What I'm suggesting is just hiding the words `Left` and `Right` via CSS, so there isn't confusion about what they mean.

**Comment by humphd** [comment]
What if you just make the `div` with id=`"working-set-list-second-pane"` be `visibility: hidden;`, so it's still there, but doesn't show at all.  It looks like this if you do:

screen shot 2015-03-12 at 1 28 04 pm

**Comment by K-LV** [comment]
So just hide the "right" one? If so, should I rename the "left" one?

**Comment by humphd** [comment]
No, that would mean a change to the Brackets code, which we don't want.  Just remove the right one from the UI with CSS, like you do with that button, and all should be well.

File: lib/hideUI.js [hunk 1]
```diff
@@ -25,6 +25,10 @@ define(function (require, exports, module) {
 	 * This function merely removes the left side tool bar
 	 */
 	function removeLeftSideToolBar() {
+		//Remove header titles
```
Comment (humphd) [inline lib/hideUI.js]: Fix this comment to reflect that we're hiding the right-split view details, since we're hijacking it.

**Comment by humphd** [comment]
One nit, r=me with that.

**Comment by sedge** [comment]
r+

**Pull request merged**